system/settings/settings.xml

```diff
@@ -1053,6 +1053,11 @@
           <default>true</default>
           <control type="toggle" />
         </setting>
+		<setting id="myvideos.handlediscn" type="boolean" label="29995" help="39198">
+		  <level>1</level>
```

Comment:
I don't think this is the correct level, as this more an expert setting -> level 3